### ISSUE TITLE
Merge or rebase with fikovnik's

### ISSUE THREAD

**Issue by karlkfi** [onsi/ShiftIt#5]
It looks like fikovnik's version has been updated a bunch. Were you planning to get these changes merged into his repo?

**Comment by onsi** [comment]
Nope. I've strayed a bunch from the original and have no time to invest in
staying uptodate.

**Comment by drsharp** [comment]
I think Fikonik's has too many options. I, for one, prefer Onsi's because it is simpler and just meets my needs. :-)  :+1:

**Comment by tgaff** [comment]
:+1:  Agree with drsharp.  I am however worried whether this fork will be maintained if @onsi doesn't have time for it.
